src/main/java/org/opennars/storage/Memory.java

```diff
@@ -60,7 +61,7 @@
  * 
  * Memory is serializable so it can be persisted and transported.
  */
-public class Memory implements Serializable, Iterable<Concept> {
+public class Memory implements Serializable, Iterable<Concept>, Resetable {
```

Comment:
Resetable has 2 t's so -it's 'resettable'

Reply:
english doesn't make any sense